src/components/v5/common/ActionSidebar/partials/VerifiedMembersTable/VerifiedMembersTable.tsx

```diff
@@ -0,0 +1,75 @@
+import clsx from 'clsx';
+import React, { FC } from 'react';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import { FormattedMessage } from 'react-intl';
+
+import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext';
+import { useMobile } from '~hooks';
+import { formatText } from '~utils/intl';
+import TableWithMeatballMenu from '~v5/common/TableWithMeatballMenu';
+import Button from '~v5/shared/Button/Button';
+
+import { useVerifiedMembersTableColumns, useGetTableMenuProps } from './hooks';
+import { VerifiedMembersTableModel, VerifiedMembersTableProps } from './types';
+
+const displayName = 'v5.common.ActionsContent.partials.VerifiedMembersTable';
+
+const VerifiedMembersTable: FC<VerifiedMembersTableProps> = ({ name }) => {
```

Comment:
could we perhaps pass in 2 functions
1. `addMember` which would wrap around form logic and add to the array
2. `removeMember` which would wrap around form logic to remove members
3. the data also needs to get passed in and not get read from the form
4. can we name it just `MembersTable` since we use it to display members we will add as verified or remove from verified